### PR TITLE
tiup: delete doc from tiup list

### PR DESCRIPTION
**Pull request by sev7ndayyoo**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
In `TiUP v1.3.1`, execute `tiup doc` will throw error `exit status 3`.
If it is based on the return result of `tiup list`, I think it should be deleted?
https://tiup.io

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by sev7ndayyoo** [comment]
@TomShawn I have no idea why the check failed > <

File: tiup/tiup-overview.md [hunk 1]
```diff
@@ -116,7 +116,6 @@ Use "tiup [command] --help" for more information about a command.
     - client：连接本机的集群
     - cluster：部署用于生产环境的集群
     - bench：对数据库进行压力测试
```
Comment (TomShawn) [inline tiup/tiup-overview.md:118]:
```suggestion
    - bench：对数据库进行压力测试 
```

**Comment by TomShawn** [comment]
> @TomShawn I have no idea why the check failed > <

Thanks very much for your contribution! Unfortunately, the CI/CD check of this PR is not running (for unknown reason), which means that this PR cannot be merged even if I approve it. I am afraid that you might need to create another PR for this change. 😕
image

**Comment by sev7ndayyoo** [comment]
> > @TomShawn I have no idea why the check failed > <
> 
> @sev7ndayyoo Thanks very much for your contribution! Unfortunately, the CI/CD check of this PR is not running (for unknown reason), which means that this PR cannot be merged even if I approve it. I am afraid that you might need to create another PR for this change. 😕
> image

OK ,I will create another PR for this.

**Pull request closed without merging**